docs/diagrams/CommentClass.puml

```diff
@@ -0,0 +1,20 @@
+@startuml
+skinparam arrowThickness 1.1
+skinparam classBackgroundColor LOGIC_COLOR
+skinparam classAttributeIconSize 0
+
+class Assignment {
+-comment: Comment
++getComment(): Comment
++commentStatus(): boolean
+}
+
+class Comment {
+-commentBody: String
+-isCommented: boolean
++getIsCommented(): boolean
++toString(): String
```

Comment:
Is toString() important? I think can omit less important details

Reply:
Is the getter method for the field